### DISCUSSION TITLE
Using custom components

### DISCUSSION THREAD

**Discussion by gushonorato** [naymspace/backpex#680] [General]
Hey, congrats on the amazing job in creating Backpex.

I have a question: is there any way to add custom components to the edit/view actions?

**Comment by Flo0807** [comment]
Hey,

can you explain what you mean by "custom components to the edit/view actions"?

You can already add additional markup to edit views by using the [Backpex slots API](https://hexdocs.pm/backpex/Backpex.LiveResource.html#c:render_resource_slot/3).

**Comment by gushonorato** [comment]
Yes, let me try to explain better. I could add some additional markup using the slots API, but is there a way to add custom markup more granularly? For example, in between fields.

**Reply by Flo0807** [reply]
Unfortunately, this is not possible at the moment.

**Comment by Flo0807** [comment]
Hey @gushonorato,

can you describe your specific use case?

And do you have ideas on how the API could look like to add custom markup more granularly?
